### ISSUE TITLE
Inconsistent minimum version of prompt-toolkit

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#144]
Between minimum-constraints.txt and requirements.txt.

This causes the manual-ci-run branch to fail on Windows, e.g. https://ci.appveyor.com/project/KSchopmeyer/pywbemtools/builds/19481239/job/7e6vauitolc3153l

```
pywbemcli --version
Traceback (most recent call last):
  File "C:\projects\pywbemtools\.tox\pywin\Scripts\pywbemcli-script.py", line 6, in <module>
    from pkg_resources import load_entry_point
  File "c:\projects\pywbemtools\.tox\pywin\lib\site-packages\pkg_resources\__init__.py", line 3019, in <module>
    @_call_aside
  File "c:\projects\pywbemtools\.tox\pywin\lib\site-packages\pkg_resources\__init__.py", line 3003, in _call_aside
    f(*args, **kwargs)
  File "c:\projects\pywbemtools\.tox\pywin\lib\site-packages\pkg_resources\__init__.py", line 3032, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "c:\projects\pywbemtools\.tox\pywin\lib\site-packages\pkg_resources\__init__.py", line 657, in _build_master
    return cls._build_from_requirements(__requires__)
  File "c:\projects\pywbemtools\.tox\pywin\lib\site-packages\pkg_resources\__init__.py", line 670, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "c:\projects\pywbemtools\.tox\pywin\lib\site-packages\pkg_resources\__init__.py", line 849, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'prompt-toolkit>=1.0.15' distribution was not found and is required by pywbemtools
```

**Comment by andy-maier** [comment]
It turns out that the `minimum-constraints.txt` file specifies the version for `prompt-toolkit` twice:
```
$ grep prompt *.txt
minimum-constraints.txt:prompt-toolkit===1.0.9
minimum-constraints.txt:prompt-toolkit===1.0.15
requirements.txt:prompt-toolkit>=1.0.15
```

**Comment by andy-maier** [comment]
Fixed in PR #146.
